src/main/kotlin/com/buildit/bookit/v1/booking/dto/Booking.kt

```diff
@@ -1,13 +1,13 @@
 package com.buildit.bookit.v1.booking.dto
 
-import java.time.ZonedDateTime
+import java.time.LocalDateTime
 
 /**
  * Booking request
  */
-data class BookingRequest(val bookableId: Int, val subject: String, val startDateTime: ZonedDateTime, val endDateTime: ZonedDateTime)
+data class BookingRequest(val bookableId: Int, val subject: String, val startDateTime: LocalDateTime, val endDateTime: LocalDateTime)
 
 /**
  * Booking response
  */
-data class Booking(val bookingId: Int, val bookableId: Int, val subject: String, val startDateTime: ZonedDateTime, val endDateTime: ZonedDateTime)
+data class Booking(val bookingId: Int, val bookableId: Int, val subject: String, val startDateTime: LocalDateTime, val endDateTime: LocalDateTime)
```

Comment:
we <might> want to return the timezone that this booking is in so that the client can calculate the correct instant... sure they could get it from the bookable (or location the bookable is in) but might be convenient here... O365 & robin both appear to do this

Reply:
adding a string for TZ here